frappe/__init__.py

```diff
@@ -466,6 +466,11 @@ def get_roles(username=None):
 	import frappe.permissions
 	return frappe.permissions.get_roles(username or local.session.user)
 
+def get_doctype_roles(doctype, access_type="read"):
```

Comment:
does this have to be part of `frappe/__init__.py` ?

Reply:
Will move to permissions.py...